garrysmod/lua/menu/problems/problem_lua.lua

```diff
@@ -77,8 +76,6 @@ function PANEL:Paint( w, h )
 	if ( count > 0 ) then
 		local txt = "x" .. count
 		surface.SetFont( "DermaMedium" )
-		local tW, tH = surface.GetTextSize( txt )
-		tW = tW
```

Comment:
The `surface.SetFont` call can be removed as well, the the `txt` variable too can just be moved into `draw.SimpleText` call.